tests/src/source/Assertion.spec.bs

```diff
@@ -11,7 +12,7 @@ namespace tests
     '+++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
 
     @it("Fail")
-    function _()
+    function _() 'bs:disable-line 1026
```

Comment:
All the functions are `function _()` which causes validation error 1026 - `Duplicate identifier`

Reply:
I think the plugin itself needs to suppress this one. Writing all the functions as `_` is a "feature" of this test framework, so it should disable those diagnostics internally so devs don't need to add this comment everywhere (defeating the entire point of naming the functions `_` in the first place)